packages/make-styles/src/renderer/createDOMRenderer.ts

```diff
@@ -114,15 +122,17 @@ export function createDOMRenderer(target: Document = document): MakeStylesDOMRen
         const css = definition[RULE_CSS_INDEX];
         const ruleCSS = dir === 'rtl' ? rtlCSS || css : css;
 
-        const sheet = getStyleSheetForBucket(bucketName, target, renderer);
-
-        try {
-          sheet.insertRule(ruleCSS, sheet.cssRules.length);
-        } catch (e) {
-          // We've disabled these warnings due to false-positive errors with browser prefixes
-          if (process.env.NODE_ENV !== 'production' && !ignoreSuffixesRegex.test(ruleCSS)) {
-            // eslint-disable-next-line no-console
-            console.error(`There was a problem inserting the following rule: "${ruleCSS}"`, e);
+        if (target) {
+          const sheet = getStyleSheetForBucket(bucketName, target, renderer);
```

Comment:
why not have `getStyleSheetForBucket` return undefined/null ?

Reply:
Then we will need to have a condition on `sheet` usage, i.e. `sheet?.insertRule`. From my POV it's less obvious 😯

Reply:
I guess it's personal preference, your call